### DISCUSSION TITLE
Platform private

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1874] [Q&A] [answered]
Hello

How can I make an organization with the platform private?

That the website is seen only by a few people.

Thank you

**Comment by sarah-beasley** [comment]
Hi, 

Just replying on here too so there is a record. By setting up a production site, this will enable the way you edit the website to be private before it is launched as a live site. Although the repositories will be open to the public, only the people you specify will be able to edit the repositories, and therefore the website.

**Comment by SandraMarq** [comment]
Hello

I will return to this topic.

I already have the platform with the configurations for a productive site.

But I want preproductive to be private, I make the repositories private but the website is not visible.

Is there a way to make the website look even if the site repository is private without using the github enterprise version?

Thank you

**Reply by sarah-beasley** [reply, marked as answer]
Hi, there is no way of doing this on GitHub as the repositories must be public for the platform to work. You would need to host on amazon or another hosting provider to be able to make your repositories private but site visible to the public.
